modules/dr_buckets/dr_buckets.tf

```diff
@@ -0,0 +1,277 @@
+variable "prefix" {
+  type        = string
+  description = "Prefix used to prepend to all object names and tags."
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "Tags to be applied to resources that support tags."
+  default     = {}
+}
+
+variable "cumulus_account_id" {
+  type = number
+  description = "Account ID of the Cumulus Account"
+}
+
+variable "aws_profile" {
+  type    = string
+  description = "AWS CLI profile to use for installation and monitoring of AWS objects."
+}
+
+## Terraform Configuration
+terraform {
+  required_version = ">= 1.5"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.0.0"
+    }
+  }
+}
+
+provider "aws" {
+  profile = var.aws_profile
+
+  ignore_tags {
+    key_prefixes = ["gsfc-ngap"]
+  }
+}
```

Comment:
This can be removed since terrform conf above already has the `required_providers`